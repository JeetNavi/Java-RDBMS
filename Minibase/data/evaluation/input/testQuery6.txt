Q(x, SUM(3)) :- R(x, y, z)
