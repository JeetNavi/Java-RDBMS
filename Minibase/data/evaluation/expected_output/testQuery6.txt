1, 9, 'adbs'
